### PR TITLE
Needs more excessive comments

### PR DESCRIPTION
**Pull request by ejay11**
Wanted to add nice things about Eric in the comments

### CODE REVIEW

File: calculator.js [hunk 1]
```diff
@@ -15,7 +15,12 @@ const divide = ((num1, num2) => {
     return num1 / num2;
 });
 
-// OPERATE FUNCION WITH SWITCH STATEMENT
+// OPERATE FUNCTION WITH SWITCH STATEMENT
+/* 
+Wow! Eric tells me this is a great way to write this function.
```
Comment (eheiberg) [inline calculator.js:20]: I never told you that. I told you this was the first transition toward something cool. Quit bad-mouthing me. My reputation is at stake.

File: calculator.js [hunk 2]
```diff
@@ -15,7 +15,12 @@ const divide = ((num1, num2) => {
     return num1 / num2;
 });
 
-// OPERATE FUNCION WITH SWITCH STATEMENT
+// OPERATE FUNCTION WITH SWITCH STATEMENT
+/* 
+Wow! Eric tells me this is a great way to write this function. 
+It's so short and simple and nice. 
+Way to go, me.
```
Comment (eheiberg) [inline calculator.js]:
```suggestion
It is so short. It is so simple and nice. 
Way to go, me - Ellen Jaquette!
```

**Review by eheiberg — COMMENTED** [review]
Hey Ellen, what's up? I was looking at your code and I saw some room for improvement. Take it or leave it. Either way, I'll approve the PR once you've addressed my comments!

File: calculator.js (hunk 1 above)
Comment (ejay11) [reply calculator.js:20]: Calm down.

File: calculator.js (hunk 2 above)
Comment (ejay11) [reply calculator.js]: I went a new route because you had a terrible suggestion. Terrible.

**Review by eheiberg — APPROVED** [review]
Yeah this is better. You were right. Thanks for telling me to calm down.

**Pull request merged**